core/controllers/custom_landing_pages_test.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for custom landing pages."""
+
+from core.tests import test_utils
+import feconf
+
+
+class FractionLandingPageTest(test_utils.GenericTestBase):
+
+    def test_fraction_landing_page_without_viewer_type(self):
+        """Test for showing the landing page for fractions,
+        without any viewer type should redirect to teacher type.
+        """
+        response = self.testapp.get(feconf.FRACTIONS_LANDING_PAGE_URL)
+        self.assertEqual(response.status_int, 302)
+        self.assertEqual(response.content_type, 'text/html')
+        response.mustcontain('/fractions_landing/teachers')
+
+    def test_fraction_landing_page_with_viewer_type(self):
+        """Test for showing the landing page for fractions,
+        with student viewer type should respond student type.
+        """
+        response = self.testapp.get(
+            '%s?viewerType=student' % (feconf.FRACTIONS_LANDING_PAGE_URL))
+        self.assertEqual(response.status_int, 200)
+        self.assertEqual(response.content_type, 'text/html')
+        response.mustcontain('/fractions_landing/student')
+
+
+class StewardsLandingPageTest(test_utils.GenericTestBase):
+    """Test for showing the landing page for stewards (parents, teachers,
+    volunteers, or NGOs).
+    """
+    def test_nonprofits_landing_page(self):
+        response = self.testapp.get(feconf.CUSTOM_NONPROFITS_LANDING_PAGE_URL)
+        self.assertEqual(response.status_int, 200)
+        self.assertEqual(response.content_type, 'text/html')
+        response.mustcontain('oppia-stewards-landing-section')
```

Comment:
This LGTM, but can you please tell me what was the issue with `Contact us to arrange a pilot with your students.`?

Reply:
Tests were passing for me locally with that, but Travis gave me a red flag. I am not sure what happened but let me guess, can it be due to having a full stop (.) or question mark(?) in the old string? I am not sure.

Reply:
@gurungrahul2, can you add a link to the file where  "Contact us to arrange a pilot with your students" is written? (I can't seem to find it!)

Reply:
I got it, a latest commit on 25 Nov changes the text thus my tests were failing.  13ab98e557

Reply:
@DubeySandeep I will adapt my changes and add a commit soon.

Reply:
done @DubeySandeep